src/java/org/apache/cassandra/cql3/Relation.java

```diff
@@ -44,7 +41,7 @@ public Operator operator()
     /**
      * Returns the list of raw IN values for this relation, or null if this is not an IN relation.
      */
-    public abstract List<? extends Term.Raw> getInValues();
+    public abstract Terms.Raw getInValues();
```

Comment:
The method `Relation#isMultiColumn` is unused, as before the patch. We can probably remove it.